views/administration/forms/form-import.hbs

```diff
@@ -31,6 +32,14 @@
         Registrierungslinks sofort nach Import an Nutzer senden
     </label>
     <p>
-        Alternativ kannst du die Registrierungslinks später abrufen und deinen Nutzern per Mail oder QR-Druckbogen zukommen lassen.
+        Alternativ kannst du die Registrierungslinks später abrufen und deinen Nutzern per Mail oder QR-Druckbogen
+        zukommen lassen.
     </p>
-</div>
+    <p class="lead">
+        ⚠️ ACHTUNG: Beim Import großer Datensätze kommt es kurz nach dem Start zu einer Fehlermeldung. Im Hintergrund
+        läuft der Importprozess trotzdem unsichtbar weiter. Zeitdauer: circa 4 Minuten pro 1000 Nutzer:innen. Bitte
+        warte die entsprechende Zeit ab und aktualisiere dann dein Browserfenster, um die aktualisierte
+        Schüler:innen-Übersicht überprüfen zu können.
```

Comment:
Maybe we sould add a hint, that we are "working" on this problem?
@dominikbruechner @konstantin-sc @sarahschaefer

Reply:
fixed,
the line is now:
[...] derzeit es kurz nach